test/workbox-build/node/lib/copy-workbox-libraries.js

```diff
@@ -0,0 +1,49 @@
+const expect = require('chai').expect;
+const path = require('path');
+const proxyquire = require('proxyquire');
+const sinon = require('sinon');
+
+const errors = require('../../../../packages/workbox-build/src/lib/errors');
+
+describe(`[workbox-build] lib/copy-workbox-libraries.js`, function() {
+  const MODULE_PATH = '../../../../packages/workbox-build/src/lib/copy-workbox-libraries';
+  const DEST_DIRECTORY = path.join(path.sep, 'test-dir');
+
+  it(`should reject with an error when the copy fails`, async function() {
+    const copyWorkboxLibraries = proxyquire(MODULE_PATH, {
+      'fs-extra': {
+        ensureDir: sinon.stub().resolves(),
+        copy: sinon.stub().rejects(),
+      },
+    });
+
+    try {
+      await copyWorkboxLibraries(DEST_DIRECTORY);
+      throw new Error('Unexpected success.');
+    } catch (error) {
+      expect(error.message).to.have.string(errors['unable-to-copy-workbox-libraries']);
+    }
+  });
+
+  it(`should resolve with the new directory name after the copy completes`, async function() {
+    const copyStub = sinon.stub().resolves();
+    const ensureDirStub = sinon.stub().resolves();
+
+    const copyWorkboxLibraries = proxyquire(MODULE_PATH, {
+      'fs-extra': {
+        copy: copyStub,
+        ensureDir: ensureDirStub,
+      },
+    });
+
+    const workboxDirectory = await copyWorkboxLibraries(DEST_DIRECTORY);
```

Comment:
This is writing to `/test`, what happens if you write to './test'?

Reply:
I've added a test case for that.